api/graphql/index.js

```diff
@@ -294,7 +294,7 @@ export function makeMutations (expressContext, userId, isAdmin, fetchOne) {
     addPostToCollection: (root, { collectionId, postId }) =>
       addPostToCollection(userId, collectionId, postId),
 
-    addRoleToMember: (root, { personId, groupRoleId, groupId }) => addRoleToMember({ personId, groupRoleId, groupId }),
+    addRoleToMember: (root, { personId, groupRoleId, groupId }) => addRoleToMember({ userId, personId, groupRoleId, groupId }),
```

Comment:
why do we have userId and personId?

Reply:
Yeah, as you got to, userId is needed to check if they are a moderator. I was just passing userId along the whole way, because that is what it was originally called. Could switch to moderatorId